packages/preview-middleware-client/tsconfig.json

```diff
@@ -5,14 +5,16 @@
     "test",
     "types"
   ],
+  "files": [
+    "./test/fixtures/api.json"
```

Comment:
Why do we need that one?

Reply:
Compiler fails when trying to import without it. However, I think its better to include json imports in a generic way.

Reply:
I am ok with it, now that I understand why.

Reply:
Fixed by [bf2ba99](https://github.com/SAP/open-ux-tools/pull/1197/commits/bf2ba99323e742ede700424cd3881a97d7633287).